app/user/spec.yaml

```
@@ -29,60 +29,31 @@ components:
           type: "object"
         hasCompletedOnboarding:
           type: "boolean"
-  parameters:
-    email:
-      name: "email"
-      description: "Email of a user profile"
-      in: path
-      required: True
-      schema:
-        type: "string"
 
 paths:
-  # User API endpoints
-  /user/{email}/profile: # TODO: change this to something other than email
-    get:
-      x-openapi-router-controller: app.user.api
-      operationId: get_user_profile
-      tags:
-        - User
-      summary: "Get details of a specified profile"
-      parameters:
-        - $ref: "#/components/parameters/email"
-      responses:
-        "200":
-          description: Return details of a specified user profile
-          content:
-            application/json:
-              schema:
-                type: object
-                items:
-                  $ref: "#/components/schemas/User"
-  /user/{email}/delete: # TODO: change this to something other than email
+  /user/delete_profile:
```

```suggestion
  /user/delete-account:
```
Best practices is hypens not underscores.